typeset/converter.js

```diff
@@ -127,59 +126,92 @@ const createMapOfMathMLElements = async (log, inputPath, cssPath, outputPath) =>
       }
 
       const id = mathNode.getAttribute('id')
-      if (percent % 10 === 0 && total > 100 && prevPercent !== percent) {
-        console.info(`Extraction Progress ${percent}%`)
+
+      // clear the inside of the element because it will be replaced once typesetting completes
+      // (maybe saves memory?)
+      // mathNode.innerHTML = ''
```

Comment:
I did something like this before but this is taking too much time. When I removed those lines time of process decreased from 26 to 3 minutes